Octokit.Tests/Clients/ReleasesClientTests.cs

```diff
@@ -21,7 +20,8 @@ public void RequestsCorrectUrl()
 
                 client.Received().GetAll<Release>(Arg.Is<Uri>(u => u.ToString() == "repos/fake/repo/releases"),
                     null,
-                    "application/vnd.github.v3");
+                    AcceptHeaders.StableVersion,
```

Comment:
Just a note on usage of ```AcceptHeaders``` helper from unit test land...

When I implemented the ```AcceptHeaders``` helper class, it was purposely only used in implementation code, and left any unit tests still needing to "hardcode" the expected headers value.  This was in order to follow the principle that the tests shouldn't just echo something from the implementation side lest there ever be a bug in the implementation side and the test just passes blindly.  Just mentioning it incase you accidentally used it here.  This would be the first case of it's usage in test land so thought it worth pointing out.

I guess I modelled it similarly to how ```RequestsCorrectUrl``` unit tests always hardcode the expected url rather than using the ```ApiUrls``` helper class like the implementation side does.  Doing a quick check now, it seems this rule has been broken for ```ApiUrls``` a couple of times, although only about 6 usages in unit tests, out of a total of 419